tests/Platform/QueryResultTypeWalkerFetchTypeMatrixTest.php

```diff
@@ -62,362 +102,4161 @@ public static function getAdditionalConfigFiles(): array
 	}
 
 	/**
-	 * @param array<string, mixed> $connectionParams
-	 * @param array<string, string|null> $expectedOnPhp80AndBelow
-	 * @param array<string, string|null> $expectedOnPhp81AndAbove
-	 * @param array<string, mixed> $connectionAttributes
-	 *
-	 * @dataProvider provideCases
+	 * @param array<string, mixed> $data
+	 * @param mixed $mysqlExpectedResult
+	 * @param mixed $sqliteExpectedResult
+	 * @param mixed $pdoPgsqlExpectedResult
+	 * @param mixed $pgsqlExpectedResult
+	 * @param mixed $mssqlExpectedResult
+	 * @param self::STRINGIFY_* $stringify
+	 *
+	 * @dataProvider provideCases
+	 */
+	public function testFetchedTypes(
+		array $data,
+		string $dqlTemplate,
+		Type $mysqlExpectedType,
+		?Type $sqliteExpectedType,
+		?Type $pdoPgsqlExpectedType,
+		?Type $pgsqlExpectedType,
+		?Type $mssqlExpectedType,
+		$mysqlExpectedResult,
+		$sqliteExpectedResult,
+		$pdoPgsqlExpectedResult,
+		$pgsqlExpectedResult,
+		$mssqlExpectedResult,
+		string $stringify
+	): void
+	{
+		$dataset = (string) $this->dataName();
+		$phpVersion = PHP_VERSION_ID;
+
+		$this->performDriverTest('pdo_mysql', self::CONFIG_DEFAULT, $data, $dqlTemplate, $dataset, $phpVersion, $mysqlExpectedType, $mysqlExpectedResult, $stringify);
```

Comment:
Each test case method should have a single `performDriverTest()` call.

Either create a new test case method for different parameters (each one will have `provideCases` data provider), or keep a single test case method, but create a carthesian product with the current `provideCases` method but add different values for drivers etc.

Reply:
Done.